packages/mapviewer/src/modules/menu/components/header/AdditionalInfoCollapsable.vue

```diff
@@ -0,0 +1,58 @@
+<script setup>
+import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+import { ref } from 'vue'
+
+import MoreInfo from '@/modules/menu/components/help/MoreInfo.vue'
+import UpdateInfo from '@/modules/menu/components/help/UpdateInfo.vue'
+
+const isOpen = ref(false)
+</script>
+
+<template>
+    <div
+        class="z-menu-1-on-hover"
```

Comment:
This z index change should maybe be applied to the collapsible menu only.
This might remove the need to track the hover state entirely (as it is shown only when `isOpen` is true)